### PR TITLE
Update zwave_js docs to reflect changes in Z-Wave JS 7.0.0

### PR DESCRIPTION
**Pull request by raman325**
## Proposed change

`Z-Wave JS 7.0.0` changes some things about the events we track. Docs must be updated accordingly

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/48094
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by MartinHjelmare — COMMENTED** [review]
We need to update the event names and data attributes.

**Comment by raman325** [comment]
> We need to update the event names and data attributes.

I thought I did? Which am I missing?

EDIT: Found it, resolved

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -149,11 +153,35 @@ These are events fired by the device using the notification command class. The `
     "node_id": 1,
     "home_id": "974823419",
     "device_id": "ad8098fe80980974",
-    "label": "Keypad lock operation",
+    "command_class": 113,
+    "command_class_name": "Notification",
+    "type_": 6,
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_js.markdown]: This is not current I think.

**Review by MartinHjelmare — COMMENTED** [review]
We need to write what each event is called now and change line 139 where the old event name is written.

File: source/_integrations/zwave_js.markdown (hunk 1 above)
Comment (raman325) [reply source/_integrations/zwave_js.markdown]: They are as far as I can tell. The command class parameters tell you whether it's coming from `Notification` or `Entry Control` and the attributes match what we get from the event

Comment (raman325) [reply source/_integrations/zwave_js.markdown]: sorry, now I see what you are saying. Additional commit incoming

Comment (MartinHjelmare) [reply source/_integrations/zwave_js.markdown]:
```suggestion
    "type": 6,
```

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -136,27 +136,53 @@ Valid code slots are between 1-254.
...
+    "event_data": "555"
+}
+```
+
 ## Scene events (Value Notification)
 
-Value Notifications are used for stateless values, like `Central Scenes` and `Scene Activation`.
+Value Notifications are used for stateless values, like `Central Scenes` and `Scene Activation`. These events fire with the `zwave_js_value_notification` event type.
 
 Value Notification example:
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_js.markdown:265]: This example needs to be updated and remove type attribute.

**Review by MartinHjelmare — APPROVED** [review]
Looks good!

**Pull request merged**